gatekeeper/Dockerfile

```diff
@@ -10,16 +10,20 @@ LABEL Name=keycloak-gatekeeper \
       Url=https://github.com/keycloak/keycloak-gatekeeper \
       Help=https://issues.jboss.org/projects/KEYCLOAK
 
-RUN adduser -D -u 1000 gatekeeper
+USER root
 
-RUN apk add --no-cache ca-certificates curl tar
+WORKDIR "/opt/gatekeeper"
 
-WORKDIR "/opt"
+RUN echo "gatekeeper:x:1000:gatekeeper" >> /etc/group
```

Comment:
@bruegth Thanks for the changes. I'm not sure about what @slaskawi and @douglaspalmer think about this. But I'd like to avoid, if possible, to have different user from the Keycloak base image. I believe would make things easier for people if we refer to a single user. See:

https://github.com/keycloak/keycloak-containers/blob/9660152ff411ee22cae03c560d0b56aafd875d53/server/tools/build-keycloak.sh#L96

In this way, we keep the references consistent with other images.

Does it make any sense?

Reply:
@abstractj @slaskawi @douglaspalmer @bruegth
+1 on keeping the images consistent across the offerings & prefer the `jboss` user similarly like it's done in the base server image.

Reply:
@abstractj @iankko I changed PR to jboss as user. But keep gatekeeper directory below /opt/jboss, please review again.

Reply:
But on other hand: Gatekeeper is based on Go and has nothing to do with `jboss` and the user has only the scope of image where he's created in.

Reply:
Also this will break 
>  One feature/change per PR

 because we change username too.

 So I prefer to revert  back to `gatekeeper `as user, and make new Ticket for the discussion to change user to `jboss`

Reply:
The user should not be jboss. This image has nothing to do with jboss. To be honest the keycloak/keycloak image should not use the user jboss either, it should have used the user keycloak.